Cargo.toml

```diff
@@ -28,7 +28,7 @@ rand = "0.7.3"
 regex = "1.3.5"
 serde = { version = "1.0", features = ["derive"] }
 serde_regex = "0.4.0"
-serde_yaml = "0.8"
+serde_yaml = "0.8.17"
```

Comment:
Two tests failed as follows:

```
```

The root problem is the old `linked-hash-map` (See https://github.com/mehcode/config-rs/issues/158 .) and this diff fixes the dependency.